macos/Sources/App/macOS/AppDelegate.swift

```diff
@@ -786,17 +795,20 @@ class AppDelegate: NSObject,
         let initialWindow: Bool
         let shouldQuitAfterLastWindowClosed: Bool
         let quickTerminalPosition: QuickTerminalPosition
+        let shouldSwitchBetweenActivationPolicies: Bool
```

Comment:
I think it's less confusing generally to keep these derived configs as dumb as possible. I recommend changing this to just copy the `macosHidden` value instead.